app/controllers/concerns/hal_actions/cache.rb

```diff
@@ -0,0 +1,34 @@
+module HalActions::Cache
+  private
+
+  def index_cache_path
+    PagedCollection.new(
+      filtered(paged(sorted(scoped(resources_base)))),
+      request,
+      item_class: self.class.resource_class,
+      item_decorator: self.class.resource_representer
+    ).cache_key
+  end
+
+  def show_cache_path
+    show_resource.updated_at.utc.to_i
```

Comment:
So this `show_resource` will have the `included` as part of the db request, yes? but all we need is the `updated_at` - perhaps overkill?